### ISSUE TITLE
TV database has zero entries, movies are fine

### ISSUE THREAD

**Issue by downingjosh** [croneter/PlexKodiConnect#556]
I have run into an issue with all of the Kodi instances on my home network.  There are zero entries in the TV shows library, movies library continues to work fine and gets new entries.  If I do a full reset of the add-on, the same thing happens (though if I set the add-on to show progress, I do see it scanning the TV shows).  I have rebooted the Plex server and uninstalled/reinstalled Kodi both 17.6 and the 18 beta and dev versions.  All AppData directories for Kodi were wiped prior to each reinstall.   I have also tried rolling back PKC to a few previous versions to no effect as well.  

The TV library in Plex works just fine on all Plex clients, I can DirectPlay in the local network and stream outside.

I enabled debug logging and see a few thousand lines (presumably for each TV show entry) which says:
`14:41:02.702 T:14524   DEBUG: PLEX.itemtypes: Adding episode with plex_id 14360
14:41:02.708 T:14524   ERROR: PLEX.itemtypes: Parent tvshow now found, skip item
14:41:02.711 T:14524   DEBUG: PLEX.itemtypes: Adding episode with plex_id 5841
14:41:02.716 T:14524   ERROR: PLEX.itemtypes: Parent tvshow now found, skip `item`

[kodi.zip](https://github.com/croneter/PlexKodiConnect/files/2536385/kodi.zip)

Any insights?

**Comment by croneter** [comment]
I am currently rewiring the library sync. This should take care of your issue, but it'll take a while. Stay tuned

**Comment by downingjosh** [comment]
The issue has been resolved; I think this may have had something to do with my Plex server.  After the Plex server (Ubuntu Server 17.10) was rebooted, it didn't mount the NFS share containing the media, which is probably why the sync failed after the reboot (I didn't stream anything after the reboot so didn't notice).  I manually mounted the NFS share last night and noticed this morning that Kodi has a full library for Movies & TV Shows.
